apps-script/documentai/documentai.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * This code sample shows how to the call the Cloud Document AI API using Apps
+ * Script with OAuth. Before running this code please see the documentation
+ * guide on getting set up with Document AI processors:
+ * https://cloud.google.com/document-ai/docs/create-processor.
+ */
+const PROJECT_ID = '';
+const PROJECT_NUMBER = '';
+const LOCATION = ''; // Format is 'us' or 'eu'
+const PROCESSOR_ID = ''; // Create processor in Cloud Console
+ 
+const TEST_FILE_ID = ''; // Drive ID of an image or pdf to use for testing
+ 
+/**
+ * Creates variables for the service account key and email address.
+ * Authorization to DocAI requires a GCP service account with a Document AI role.
+ */
+// Sets variable for service account privat3e key.
+var PRIVATE_KEY = '';
+ 
+// Sets variable for service account email address.
+var CLIENT_EMAIL = 'YOUR-SERVICE-ACCOUNT@appspot.gserviceaccount.com';
+ 
+function processDocument(docBytes) {
+    var service = getService();
```

Comment:
Inconsistent formatting (indentation)

Reply:
I should really add eslint in this subdirectory